docs/ref/models/instances.txt

```diff
@@ -749,9 +749,9 @@ This template code is much better:
     <a href="{{ object.get_absolute_url }}">{{ object.name }}</a>
 
 The logic here is that if you change the URL structure of your objects, even
-for something simple such as correcting a spelling error, you don't want to
-have to track down every place that the URL might be created. Specify it once,
-in ``get_absolute_url()`` and have all your other code call that one place.
+for something like correcting a spelling error, you don't want to have to track
```

Comment:
I think "for something like correcting a spelling error" is *implying* that this is a simple task without saying it, and it's better to have the word in there then. Perhaps "small" rather than "simple" ?